general.go

```diff
@@ -104,3 +122,68 @@ func (c *Client) Close() error {
 
 	return nil
 }
+
+// PerformAction will request the connected and paired nuki opener to perform the given command.
```

Comment:
As far as I see this is not only for opener but also for smart lock...

Reply:
yeah thats correct :innocent: